src/test/java/com/puppycrawl/tools/checkstyle/checks/blocks/EmptyBlockCheckTest.java

```diff
@@ -148,7 +148,7 @@ public void testInvalidOption() throws Exception {
         try {
             final String[] expected = CommonUtils.EMPTY_STRING_ARRAY;
 
-            verify(checkConfig, getPath("InputEmptyBlockDefault.java"), expected);
+            verify(checkConfig, getPath("InputEmptyBlockSemantic.java"), expected);
```

Comment:
this is probably reason for weird changes below.
You changed `InputEmptyBlockDefault` -> `InputEmptyBlockSemantic` and then changed `InputEmptyDefault` to same name `InputEmptyBlockDefault`.

Leave `InputEmptyBlockDefault` as it originally was.
Rename `InputEmptyDefault` to some other unique name.